final_task/rss_reader/Handler.py

```diff
@@ -0,0 +1,107 @@
+import feedparser
+from dataclasses import dataclass, asdict
+import html
+
+from Log import log_decore
+
+
+# convert from News class to json
```

Comment:
Docstrings should be like this: `"""Text"""` first line in the function/class/method/module